internal/openshift/client/openshift_client.go

```diff
@@ -143,6 +145,56 @@ func (o openShift) Idle(apiURL string, bearerToken string, namespace string, ser
 	return
 }
 
+// Reset deletes a pod and start a new one
+func (o openShift) Reset(apiURL string, bearerToken string, namespace string) (err error) {
```

Comment:
for a single return, please use `return err` instead of `return` and since that is what you are doing anyway `(err error)` can become `error`